### ISSUE TITLE
Training on SDXL

### ISSUE THREAD

**Issue by Revive624** [g-luo/dual_process#4]
Very good work!
I tried it on finetuning SDXL. However, the loss is hard to reduce. Have you ever tried SDXL or do you have any idea on possible solutions.
Thank you!

**Comment by rayray9999** [comment]
Hello, I've also tried on SDXL and met the same problem. May I ask if you have found a way to solve it?
